src/components/og-spinner/og-spinner.tsx

```diff
@@ -0,0 +1,49 @@
+/**
+ * ORGENIC-UI
+ * @license MIT
+ * See LICENSE file at https://github.com/orgenic/orgenic-ui/blob/master/LICENSE
+ **/
+
+import { h,  Component, Prop, Host } from '@stencil/core';
+
+const sizes = {
+  s: '16px',
+  m: '32px',
+  l: '48px'
+}
```

Comment:
Just an idea: Wouldn´t it make sense to just apply a 'sizing-class' to the host (e.g. `small` etc.) depending on the `size` property, and use this class to size the SVG with the help of CSS custom properties (e.g. `--og-spinner-Size--small` etc.)? This would increase the styling/sizing capabilities for the developer.

Reply:
Changed it to behave like our progressbar